Doc/library/typing.rst

```diff
@@ -32,6 +32,10 @@ In the function ``greeting``, the argument ``name`` is expected to be of type
 :class:`str` and the return type :class:`str`. Subtypes are accepted as
 arguments.
 
+New features have been added to the typing module in each version of Python.
```

Comment:
```suggestion
New features are frequently added to the ``typing`` module.
```
So this doesn't go out of date if we don't end up adding anything to 3.12 or something.